addons/spreadsheet/static/src/pivot/pivot_model.js

```diff
@@ -289,7 +289,15 @@ export class SpreadsheetPivotModel extends PivotModel {
         const { cols, rows } = this._getColsRowsValuesFromDomain(domain);
         const group = JSON.stringify([rows, cols]);
         const values = this.data.measurements[group];
-        return (values && values[0][measure]) ?? "";
+
+        if (
+            values &&
+            Object.prototype.hasOwnProperty.call(values[0], measure) &&
```

Comment:
what is this ?

Reply:
It checks if `values[0]` has a property called `measure`.

Previously, I used `values[0].hasOwnProperty`, but encountered an ESLint error while playing around with tests: `Do not access Object.prototype method 'hasOwnProperty' from target object no-prototype-builtins`.